ourjapanlife/findadoc-server
# Dockerize Postgres

Resolves #66 👹 

- [x] run postgres in docker
- [x] get the migrations to run
- [x] Ensure migrations run from outside the container, e.g. `yarn migrate`

## Testing instructions

1. Check out the readme changes and follow the instructions to see if you get a container working
2. Run `yarn migrate` from your localhost (not inside docker) to ensure there is a seamless connection between your local environment and the dockerized db
3. Try `yarn prisma studio` to ensure there is a seamless connection between the explorer and the dockerized db
